exercise_03.py

```diff
@@ -0,0 +1,20 @@
+def foot_to_meters(foot):
+    meter = foot * 0.305
+    return "{0:.3f}".format(meter)
+
+
+def meters_to_foot(meter):
+    foot = meter / 0.305
```

Comment:
Again, this variable `foot` is needless, you can achieve the same by replacing it with what you are assigning in this line. My point is you should create a variable when you are going to use that variable at least more than once. Otherwise, it is just a needless binding you are creating in your program.